tests/languages/rescript/class-name_feature.test

```diff
@@ -0,0 +1,35 @@
+Belt.Int.toString(n)
+<Button />
+@react.component
+type poly = [ | #Poly | #Variant ]
+
+----------------------------------------------------
+
+[
+	["class-name", "Belt"],
+	".",
+	["class-name", "Int"],
+	["function", ".toString"],
+	"(n)\r\n",
+
+	["operator", "<"],
+	["class-name", "Button"],
+	["operator", "/"],
+	["operator", ">"],
+
+	["class-name", "@react.component"],
+
+	["keyword", "type"],
+	["constant", "poly"],
+	["operator", "="],
+	" [ ",
```

Comment:
Could we please highlight those as punctuation? Just add a `'punctuation': /[(){}[\],;.]/` after `operaor`.

Reply:
Sure

Reply:
Done